scripts/utils/price_utils.js

```diff
@@ -131,6 +131,7 @@ const getOneinchPrice = async function (baseToken, quoteToken, globalPriceStorag
     throw new Error("Invalid token input for retrieving price from aggregator.")
   }
   const quoteTokenAmount = toErc20Units("1", quoteToken.decimals)
+  console.log(`Requesting ${baseToken.symbol}-${quoteToken.symbol} price from external source: 1Inch`)
```

Comment:
Maybe we want to remove this?

Reply:
You don’t want to know when an external price source is queried? I thought it was good to know. Especially for that user whose script failed after 750 seconds. It was likely due to this request. Then he would have known.

Reply:
I am indifferent, but I think that it spams the inexperienced user with too much information. They won't be able to make this mentioned connection
A cleaner way to keep it,  might be to enable it in debug mode.

Reply:
OK I will remove it.

Reply:
Mostly because this method does have a debug optional parameter.